use/httpserver.rst

```diff
@@ -24,20 +24,22 @@ The HTTP comes with a couple of additional variables:
 Using the server
 ================
 
-At the root of the server, you will find a list of the plugins that have a Web
-interface, and a link to them. Each plugin has its own subdirectory(ies).
+At the root of the server, you will find a list of the plugins that
+have a Web interface, and a link to them. Each plugin has one or more
+subdirectories of its own.
 
-You may also want to have Apache in front of Supybot's HTTP server, if 
-you want to use subdomains. Here is an example of configuration 
-(I didn't test it with the rewrite, please notify me whether it works or 
-not):
+You may also want to run Apache httpd or Nginx in front of Supybot's HTTP
+server, if you want to use subdomains or load balancing.
+
+Here is an example of Apache httpd configuration (I didn't test it
+with the rewrite, please notify me whether it works or not):
 
 .. code-block:: apache
 
     <VirtualHost 0.0.0.0:80>
         ServerName stats.yourdomain.org
         <Location />
-                ProxyPass http://localhost:8080/webstats/
+                ProxyPass HTTP://localhost:8080/webstats/
```

Comment:
I've never seen a scheme written in upper-case.
Any particular reason for using it?